src/components/SimpleTodo/TodoForm.js

```diff
@@ -20,16 +23,36 @@ submitHandler=(event)=>{
         text:this.state.text,
         complete:false
     })
+      
     this.setState({
         text:''
```

Comment:
make this onto one line if you are not updating more than one object for the state